python/src/deltachat/account.py

```diff
@@ -7,14 +7,15 @@
 except ImportError:
     from Queue import Queue
 
+import deltachat
 from . import capi
 from .capi import ffi
-import deltachat
+from .myattr import attrib_CData, attrs, attrib_int, cached_property
```

Comment:
I'm not a huge fan of the attrib_* shortcuts, they're not very verbose to write and the shortcuts hide things somewhat.  Likewise importing attris/attrib via myattr is less clear then using `attr.s` `attr.ib` directly, now I have to worry about whether they're actually what I think they'll be which wouldn't be the case if importing `attr` directly.

I'm generally not a fan of importing objects directly btw, always prefer importing modules and accessing objects inside them via the module.  But clearly you don't share that view so I don't mind, just thought I'd mention it.

Anyway, all this is a style decision really, I've argued the readability point of view but up to you in the end.